ArduPlane/Parameters.h

```diff
@@ -343,6 +343,7 @@ class Parameters {
         k_param_dspoiler_rud_rate,
 
         k_param_logger = 253, // Logging Group
+        k_param_precland = 256,
```

Comment:
I'd rather see this in g2 instead of going over 255

Reply:
Is there documentation/example anywhere on how to do the g2 list? Not sure how to implement this. Doesn't look as simple I thought it would be